src/Avalonia.Base/Animation/Spring.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System;
+using System.ComponentModel;
+using System.Globalization;
+using Avalonia;
+using Avalonia.Utilities;
+
+namespace Avalonia.Animation;
+
+/// <summary>
+/// Determines how an animation is used based on spring formula.
+/// </summary>
+[TypeConverter(typeof(SpringTypeConverter))]
+public class Spring : AvaloniaObject
```

Comment:
Are you sure that it should be AvaloniaObject?

Reply:
Remove AvaloniaObject usage